### DISCUSSION TITLE
Page through issue List which has more than 100 entries

### DISCUSSION THREAD

**Discussion by ElCaptaine** [ankitpokhrel/jira-cli#674] [General]
Hey there,
is it possible to change the "page". At the moment I've got the problem that some of my statements return more than 100 entries, and I can't change the pagination size to more than 100 entries per page. So i'am some what stuck on the first "page". I didn't found any help in the documentation. Is there a key mapping to load the other entries (entries > 100)?

Thanks in advanced for your help

**Comment by ankitpokhrel** [comment]
Hi @ElCaptaine, `paginate` option available when listing issues

```
$ jira issue list --help
...
 --paginate string         Paginate the result. Max 100 at a time, format: <from>:<limit> where <from> is optional (default "0:100")
...
```

**Reply by ElCaptaine** [reply]
HI @ankitpokhrel 
Okay, I got it. But how do I skip the page in the terminal `Showing 100 of 525 results for project "Project"`.
BC if I do `jira issue list --pagination "int:int"` I just can list all issue from 1 to 100 but not further...
So how to change the page.

Thanks by the way for your help

**Reply by ElCaptaine** [reply]
or to be more precise, how can I fetch the next issues

**Comment by stan928** [comment]
I’d like this too

**Comment by mmllc-jsilverman** [comment]
Just another "me too"; it's 2 years later -- does this work?

No matter how I phrase the arguments, I only ever get the first 100 items.

What is the "<from>" argument supposed to do? Is it "From item number X" or is it "From page number X"? Either way, it appears to be ignored.

In fact, the <from> argument seems to be ignored even if the <limit> argument is lower than the fixed maximum. I *only* ever get the first X rows.

I'm using jira-cli v1.7.0 (Aug 2025)
